feature_tests.rb

```diff
@@ -0,0 +1,10 @@
+Dir["./lib/*.rb"].each { |file| require file }
+
+interface = CustomerInterface.new
+dish = Dish.new("Katsu Chicken Curry", 7.99, 20)
+another_dish = Dish.new("Sushi Platter", 21.99, 30)
+
```

Comment:
At the moment, the interface contains the menu but then, to order things, Dishes are created outside of that.  Something doesn't quite feel right there.  I.e. Should there be a more explicit connection between Dishes and the menu?

Reply:
I wonder whether the menu should be a collection of Dishes